utils/slither_format/format_naming_convention.py

```diff
@@ -0,0 +1,600 @@
+import re, sys, logging
+from slither.core.expressions.identifier import Identifier
+from slither.core.cfg.node import Node
+from slither.slithir.operations import NewContract
+from slither.slithir.operations import Member
+from slither.utils.colors import red, yellow, set_colorization_enabled
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('Slither.Format')
+set_colorization_enabled(True)
+
+class FormatNamingConvention:
```

Comment:
Consider adding code documentation around the string manipulation and the use of regex